mender-client-acceptance-testing/Dockerfile

```diff
@@ -16,10 +16,26 @@ ENV LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8
 COPY requirements-py3.txt .
 RUN pip3 install -r requirements-py3.txt
 
+# For warrior tests
+RUN apt-get install -y python-pip && \
+    apt-get install -qqy python-pip && \
+    pip2 install --upgrade pip && \
+    pip2 install requests --upgrade && \
+    pip2 install pytest --upgrade && \
+    pip2 install filelock --upgrade && \
+    pip2 install pytest-xdist --upgrade && \
+    pip2 install pytest-html --upgrade && \
+    pip2 install -I fabric==1.14.0 && \
+    pip2 install psutil --upgrade && \
+    pip2 install boto3 --upgrade && \
+    pip2 install pycrypto --upgrade && \
```

Comment:
personally I would move these to a `requirements.txt` and install from there, with the `wget` / `pip2 install` combination using a `wget -O meta-mender-requirements.txt https://raw.git...` but that is just personal taste I guess...

Reply:
@mzedel I understand. I made this explicit following exactly how it was for `warrior` up until few weeks ago. `warrior` is a branch in not too good shape (Python 2, requirements.txt incomplete) and I'll rather let it be for the few more months that has to be maintained. We'll declare it end of like April 2021 and remove these lines from here :)

Reply:
@mzedel Well, thinking about this again... Maybe I fix this anyway (independently of the current PR and work) to be able in the future to rescue the branch and do some potential PS deal or something. Low priority, I'll think about it.